themes/default/content/registry/packages/docker/buildx/--index/_index.md

```diff
@@ -0,0 +1,1310 @@
+
+---
+title: "Index"
+title_tag: "docker.buildx.Index"
+meta_desc: "Documentation for the docker.buildx.Index resource with examples, input properties, output properties, lookup functions, and supporting types."
+layout: api
+no_edit_this_page: true
+---
+
+
+
+<!-- WARNING: this file was generated by Pulumi Docs Generator. -->
+<!-- Do not edit by hand unless you're certain you know what you are doing! -->
+
+An index (or manifest list) referencing one or more existing images.
+
+Useful for crafting a multi-platform image from several
+platform-specific images.
+
+This creates an OCI image index or a Docker manifest list depending on
+the media types of the source images.
```

Comment:
Is this required to publish multi-platform images with `docker.buildx.Image`?

Reply:
No, you can publish multi-platform just fine but they don't play well with caching https://github.com/docker/buildx/discussions/1382. If you try to cache a multi-platform image you'll get a warning telling you to use this instead.